sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ComponentDescriptorFactory.scala

```diff
@@ -131,6 +152,14 @@ private[impl] trait ComponentDescriptorFactory {
    * Inspect the component class (type), validate the annotations/methods and build a component descriptor for it.
    */
   def buildDescriptorFor(componentClass: Class[_], nameGenerator: NameGenerator): ComponentDescriptor
+
+  private[impl] def buildJWTOptions(method: Method): Option[MethodOptions] = {
+    if (hasJwtMethodOptions(method)) {
+      Some(kalix.MethodOptions.newBuilder().setJwt(jwtMethodOptions(method)).build())
+    } else {
+      None
+    }
```

Comment:
Here we can use `Option.when`

```suggestion
    Option.when(hasJwtMethodOptions(method)) {
      kalix.MethodOptions.newBuilder().setJwt(jwtMethodOptions(method)).build()
    }
```